third_party/dartdoc/README.md

```diff
@@ -0,0 +1,4 @@
+# Dartdoc resource files
+
+Vendoring resource files from:
+https://github.com/dart-lang/dartdoc/tree/master/lib/resources
```

Comment:
Let's reference the `update-dartdoc-resources.sh` script here.

Reply:
Will do in a follow up.